docs/DeveloperGuide.md

```diff
@@ -38,8 +38,38 @@
     + [Non-Functional Requirements](#non-functional-requirements)
     + [Glossary](#glossary)
 6. [Appendix B: Instructions for manual testing](#appendix-b-instructions-for-manual-testing)
-   + [Launch and Shutdown](#launch-and-shutdown)
-   
+    + [Launch and Shutdown](#launch-and-shutdown)
+
+## Delete Module {#delete-module}
+
+The DeleteModule functionality allows users to remove a module from the ModuleList.
+It is facilitated by DeleteModuleCommand class which is an extension of the Command class.
+Below are the steps to recreate it
+
+Step 1: Define the Constructor :
+The constructor of the DeleteModuleCommand class takes in a moduleCode as a parameter. This moduleCode is used to find
+the module that needs to be deleted from the ModuleList.
+
```

Comment:
Perhaps you could refer to https://se-education.org/addressbook-level3/DeveloperGuide.html#proposed-undoredo-feature to describe the implementation with an example